packages/ia-components/sandbox/theatres/with-youtube-spotify/audio-with-youtube-spotify-main.jsx

```diff
@@ -162,9 +162,8 @@ class AudioPlayerWithYoutubeSpotify extends Component {
       audioSource = {};
       audioSource[channelToPlay] = albumSource;
     }
-    const trackToHighlight = trackSelected === null && trackStartingPoint !== null ? trackStartingPoint : trackSelected;
```

Comment:
@koderjoker what happens to `trackToHighlight`?

Reply:
The value of trackToHighlight (which is only sent to the TheatreTrackList) isn't affected since it's calculated in render.

Reply:
what about line `163` where it is setting `audioSource` - should we return early there so this next line doesn't mutate the value?